recipe/meta.yaml

```diff
@@ -16,28 +16,26 @@ source:
   - git_url: https://skia.googlesource.com/skcms
     git_rev: 42030a771244ba67f86b1c1c76a6493f873c5f91
     folder: third_party/skcms
+  # They pull in 3 headers from here and copy them into jpegli
   - url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/{{ jpeg_turbo_version }}.tar.gz
     sha256: ed729127097af5000bfeddc6b3e679c4363bb1618ebfc07efb6b47f576471b4b
     folder: third_party/libjpeg-turbo
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - pkg-config
-    - cmake >=3.10
+    - cmake
     - make  # [not win]
   host:
     - brotli
     - libhwy
-    # lcms2 requires jpeg9, which is incompatible
-    # - lcms2
+    - lcms2
```

Comment:
Looks like the global pinning uses the wrong name for this (https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/0610eae8c4c895f20e0626acf041ca68674f9ee5) - should be
```diff
-lcms:
+lcms2:
   - 2
```

Reply:
Good catch!!! The nice thing is that 2.16 is still being pulled in even without the global pin

 2.16-hb7c19ff_0